internal/stacktrace/stacktrace.go

```diff
@@ -0,0 +1,195 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016 Datadog, Inc.
+
+//go:generate msgp -o=stacktrace_msgp.go -tests=false
+
+package stacktrace
+
+import (
+	"os"
+	"runtime"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/go-secure-stdlib/parseutil"
+)
+
+var enabled = true
+var defaultTopFrameDepth = 8
+var defaultMaxDepth = 32
+
+const defaultCallerSkip = 3
+const stackTraceDepthEnvVar = "DD_APPSEC_MAX_STACK_TRACE_DEPTH"
+const stackTraceDisabledEnvVar = "DD_APPSEC_STACK_TRACE_ENABLE"
```

Comment:
Would match the style of other things in `dd-trace-go`

```suggestion
var (
	enabled = true
	defaultTopFrameDepth = 8
	defaultMaxDepth = 32
)

const (
	defaultCallerSkip = 3
	stackTraceDepthEnvVar = "DD_APPSEC_MAX_STACK_TRACE_DEPTH"
	stackTraceDisabledEnvVar = "DD_APPSEC_STACK_TRACE_ENABLE"
)
```